pkg/config/derive_test.go

```diff
@@ -0,0 +1,298 @@
+package config
+
+import (
+	"os"
+	"reflect"
+	"testing"
+
+	acsapi "github.com/openshift/openshift-azure/pkg/api"
+	"github.com/openshift/openshift-azure/pkg/util/fixtures"
+)
+
+func Test_derived_ImageResourceGroup(t *testing.T) {
+	tests := []struct {
+		want string
+	}{
+		{
+			want: "test-image1",
+		},
+		{
+			want: "test-image2",
+		},
+	}
+	defer os.Setenv("IMAGE_RESOURCEGROUP", os.Getenv("IMAGE_RESOURCEGROUP"))
+
+	for _, tt := range tests {
+		t.Run(tt.want, func(t *testing.T) {
+			d := derived{}
+			os.Setenv("IMAGE_RESOURCEGROUP", tt.want)
+			if got := d.ImageResourceGroup(); got != tt.want {
+				t.Errorf("derived.ImageResourceGroup() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func Test_derived_ImageConfigFormat(t *testing.T) {
```

Comment:
Don't test this, we need to revert its move in derive, see https://github.com/openshift/openshift-azure/issues/487

Reply:
Okay

Reply:
https://github.com/openshift/openshift-azure/pull/503